src/constants.ts

```diff
@@ -0,0 +1,25 @@
+const daysNumberOnTimeGrid: number = 7;
+const hoursInDay: number = 24;
+const minutesInHour: number = 60;
+const daysInWeek: number = 7;
+const cellHeightInPx: number = 49;
+const currentEventTileId: string = "current-event-tile";
+const defaultEventLengthInHours: number = 1;
+const emptyEventTitle: string = "(no title)";
+const timeStepInMinutes: number = 15;
+const modalInputsIds: { [key: string]: string } = { date: "date", timeStart: "time-start", timeEnd: "time-end" };
```

Comment:
For object constants the workflow should be the following:

```suggestion
const modalInputsIds = { 
  date: "date", 
  timeStart: "time-start", 
  timeEnd: "time-end" 
} as const;
```

<img width="886" alt="Screenshot 2024-08-08 at 11 01 27" src="https://github.com/user-attachments/assets/cb98d0dc-3a06-4e17-9860-5d144e3fa479">

<img width="1105" alt="Screenshot 2024-08-08 at 11 02 04" src="https://github.com/user-attachments/assets/08eca8cd-4a16-4aaa-b3a6-76032ea2087e">

Reply:
The approach above will allow you do define object as you want. Later on mark it as a const and afterwards you can even generate a type for it using:

```
type ObjectValues<T> = T[keyof T];
type ModalInputId = ObjectValues<typeof modalInputsIds>
```